auth-service/src/test/kotlin/uk/gov/communities/delta/controllers/AdminEditUserControllerTest.kt

```diff
@@ -377,6 +372,7 @@ class AdminEditUserControllerTest {
                             controller,
                             mockk(relaxed = true),
                             mockk(relaxed = true),
+                            mockk(relaxed = true),
```

Comment:
Really minor, but slightly annoying we have to do this. 

Is it any neater to give the bearerTokenRoutes function default values of mockk(relaxed = true) for all the parameters? Not great to have something that's purely for testing in the real app code, but it would be nicer to just do bearerTokenRoutes(adminEditUserController = controller)

Reply:
https://github.com/communitiesuk/delta-auth-service/pull/102